docs/tutorials/simple_trading.py.md

```diff
@@ -0,0 +1,142 @@
+# Tutorial: Simple Futures Trading
+
+In this tutorial you will learn the basics of how to use our Python API. It will cover:
+* subscribing to our realtime WebSocket market stream with public trading data (order books, trades, etc.)
+* connecting to our realtime WebSocket user stream to subscribe to your private data (your account changes, order 
+  updates, etc.) and to send commands to the exchange (place/cancel orders, etc.)
+* making basic trading decisions in reaction to received data
+
+## 1. Imports
+
+In order to use Quedex API, we need to import basic data structures, stream factories 
+and twisted reactor.
+
+```python
+from quedex_api import (
+  Exchange,  
+  MarketStream, 
+  MarketStreamListener, 
+  MarketStreamClientFactory,
+  Trader,
+  UserStream,
+  UserStreamListener,
+  UserStreamClientFactory,
+  )
+
+from twisted.internet import reactor
+
+from time import time
+```
+
+## 2. Trading Strategy
+
+Next, we will define user-supplied listeners which will be later passed to `UserStream` and `MarketStream` and which
+will receive messages that arrive on the WebSockets. For the sake of example, our `MarketStreamListener` will implement
+a dummy trading strategy which will trade the first futures instrument it finds (cf. `on_instrument_data`) and will
+place a sell order whenever the ask is higher than 0.001 BTC per USD (cf. `on_order_book`; remember that the contracts 
+have USD as underyling and prices are expressed in BTC! i.e. this corresponds to buying when BTCUSD is lower than 1000).
+
+// TODO: really? what about threading? does twister guarantee that? 
+Notice that we are guaranteed to know the `selected_instrument_id` before we receive any order book - `instrument_data`
+is the first message that arrives on market stream (cf. documentation of `MarketStreamListener`).
```

Comment:
All callbacks are called in the main thread as per [this document](http://twistedmatrix.com/documents/current/core/howto/threading.html). I've confirmed that experimentally.

BTW, twisted uses epoll on Linux - seems to be a nice piece of software.